libr/util/print.c

```diff
@@ -935,7 +936,12 @@ R_API void r_print_hexdump(RPrint *p, ut64 addr, const ut8 *buf, int len, int ba
 					r_print_cursor (p, j, 1);
 					// stub for colors
 					if (p && p->colorfor) {
-						a = p->colorfor (p->user, n, true);
+						int type = r_core_anal_address (user, addr + j);
```

Comment:
Remove this and use p->iob.map_get (not implemented), but should return true or false depending if its mapped. you already have is_valid_address() or stgh like this,